image_spec.rb

```diff
@@ -32,99 +32,99 @@
+      expect { browser.image(index: 1337).id }.to raise_error(Watir::Exception::UnknownObjectException)
     end
   end
 
   describe "#src" do
     it "returns the src attribute of the image if the image exists" do
-      expect(browser.image(:id, 'square').src).to include("square.png")
+      expect(browser.image(id: 'square').src).to include("square.png")
     end
 
     it "returns an empty string if the image exists and the attribute doesn't" do
-      expect(browser.image(:index, 0).src).to eq ""
+      expect(browser.image(index: 0).src).to eq ""
     end
 
     it "raises UnknownObjectException if the image doesn't exist" do
-      expect { browser.image(:index, 1337).src }.to raise_error(Watir::Exception::UnknownObjectException)
+      expect { browser.image(index: 1337).src }.to raise_error(Watir::Exception::UnknownObjectException)
     end
   end
 
   describe "#title" do
     it "returns the title attribute of the image if the image exists" do
-      expect(browser.image(:id, 'square').title).to eq 'Square'
+      expect(browser.image(id: 'square').title).to eq 'Square'
     end
 
     it "returns an empty string if the image exists and the attribute doesn't" do
-      expect(browser.image(:index, 0).title).to eq ""
+      expect(browser.image(index: 0).title).to eq ""
     end
 
     it "raises UnknownObjectException if the image doesn't exist" do
-      expect { browser.image(:index, 1337).title }.to raise_error(Watir::Exception::UnknownObjectException)
+      expect { browser.image(index: 1337).title }.to raise_error(Watir::Exception::UnknownObjectException)
     end
   end
 
   describe "#respond_to?" do
     it "returns true for all attribute methods" do
-      expect(browser.image(:index, 0)).to respond_to(:class_name)
-      expect(browser.image(:index, 0)).to respond_to(:id)
-      expect(browser.image(:index, 0)).to respond_to(:style)
-      expect(browser.image(:index, 0)).to respond_to(:text)
+      expect(browser.image(index: 0)).to respond_to(:class_name)
+      expect(browser.image(index: 0)).to respond_to(:id)
+      expect(browser.image(index: 0)).to respond_to(:style)
+      expect(browser.image(index: 0)).to respond_to(:text)
     end
   end
 
   # Manipulation methods
   describe "#click" do
     it "raises UnknownObjectException when the image doesn't exist" do
-      expect { browser.image(:id,    'missing_attribute').click }.to raise_error(Watir::Exception::UnknownObjectException)
-      expect { browser.image(:class, 'missing_attribute').click }.to raise_error(Watir::Exception::UnknownObjectException)
-      expect { browser.image(:src,   'missing_attribute').click }.to raise_error(Watir::Exception::UnknownObjectException)
-      expect { browser.image(:alt,   'missing_attribute').click }.to raise_error(Watir::Exception::UnknownObjectException)
+      expect { browser.image(id:    'missing_attribute').click }.to raise_error(Watir::Exception::UnknownObjectException)
```

Comment:
Everywhere else seems to have single space after colon, instead of matching the spacing.